scripts/release.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+atlas_api="https://atlas.hashicorp.com/api/v1/box/hashbang/shell-server"
+atlas_token=$(jq -r '.atlas_token' config.json)
+bintray_user=$(jq -r '.bintray_user' config.json)
+bintray_key=$(jq -r '.bintray_key' config.json)
+version="$VERSION"
```

Comment:
It's not important to have this, but it would mean the code is a few lines shorter.